contents.json

```diff
@@ -2323,7 +2323,15 @@
       "description": "Server-side Swift. The Perfect library, application server, connectors and example apps.",
       "homepage": "https://github.com/PerfectlySoft/Perfect",
       "tags": ["linux"]
-    }, {
+    },
+    { 
+      "title": "SwiftEngine",
+      "category": "webserver",
+      "description": "Apple Swift based HTTP server. The answer for a Swift based, turn key, crash resilient, high scale, and production grade web server.",
```

Comment:
@hamletazarian could you remove "Swift" from the description, according to the following rule: "Description does not say "written in Swift" or variant"?

Reply:
I've been trying to find the reasoning behind this rule in the project documentation. Is there a particular reason for it?

Please note: I'm not trying to start a flame-war or anything. Just really curious about the reason for it.

Reply:
@clsk because is obvious that if you are listed here it's something related to swift, and if we can get lightweight the json file the better it is.
Otherwise you will have swift in, at least, 2 fields for all entry...